tests/integration/integration_test_case.py

```diff
@@ -2,18 +2,50 @@
 import re
 import unittest
 
+from bs4 import BeautifulSoup
+
 from app import create_app
 from app import settings
-from app.data_model.database import QuestionnaireState
+from app.data_model.database import QuestionnaireState, EQSession, UsedJtiClaim
+from app.submitter.submitter import Submitter, SubmitterFactory
 
 from tests.integration.create_token import create_token
 
 
-class IntegrationTestCase(unittest.TestCase):
+class MyMockSubmitter(Submitter):
+
+    def __init__(self):
+        self._message = None
+        self._submitted_at = None
+
+    @staticmethod
+    def encrypt_message(message):
+        return message
+
+    def get_message(self):
+        return self._message
+
+    def send_message(self, message, queue):
+        self._message = message
+        return True
+
+
+class IntegrationTestCase(unittest.TestCase):  # pylint: disable=too-many-public-methods
 
     def setUp(self):
-        settings.EQ_SERVER_SIDE_STORAGE_DATABASE_URL = "sqlite://"
+        # Cache for requests
+        self.last_url = None
+        self.last_response = None
+        self.last_csrf_token = None
 
+        # Perform setup steps
+        self._setUpSettings()
+        self._setUpMockSubmitter()
+        self._setUpApp()
+
+    @staticmethod
+    def _setUpSettings():
```

Comment:
yeah tricky as the underlying base class (`TestCase`) uses that format but PEP8 is `_` based; so I think the `Assert` calls should be consistent with TestCase but not sure about these ones; as `setUp` is from `TestCase` but the private functions could be `_` based?